packages/react-table/src/components/TableComposable/Td.tsx

```diff
@@ -62,10 +62,14 @@ export interface TdProps extends BaseCellProps, Omit<React.HTMLProps<HTMLTableDa
   isStickyColumn?: boolean;
   /** Adds a border to the right side of the cell */
   hasRightBorder?: boolean;
+  /** Adds a border to the left side of the cell */
+  hasLeftBorder?: boolean;
   /** Minimum width for a sticky column */
   stickyMinWidth?: string;
   /** Left offset of a sticky column. This will typically be equal to the combined value set by stickyMinWidth of any sticky columns that precede the current sticky column. */
   stickyLeftOffset?: string;
+  /** Right offset of a sticky column. This will typically be equal to the combined value set by stickyMinWidth of any sticky columns that precede the current sticky column. */
```

Comment:
Based on the example, it looks like this description should be something like:

```suggestion
  /** Right offset of a sticky column. This will typically be equal to the combined value set by stickyMinWidth of any sticky columns that come after the current sticky column. */
```